### PR TITLE
improve localStorage for better user experience

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
fix issue #5127
- store columns use this format `experimentID_paraColumns`(for example)
- don't delete localstorage
- website only read the right storage
#### Test Options ####
  - [x] full test

### CODE REVIEW

**Review by liuzhe-lz — APPROVED** [review]

File: ts/webui/src/components/experiment/trialdetail/chart/Para.tsx [hunk 1]
```diff
@@ -56,9 +56,9 @@ class Para extends React.Component<ParaProps, ParaState> {
             customizeColumnsDialogVisible: false,
             availableDimensions: [],
             chosenDimensions:
-                localStorage.getItem('paraColumns') !== null
+                sessionStorage.getItem('paraColumns') !== null
```
Comment (ultmaster) [inline ts/webui/src/components/experiment/trialdetail/chart/Para.tsx:59]: If I run a different experiment on the same port, would the storage still be persistent?

Comment (Lijiaoa) [reply ts/webui/src/components/experiment/trialdetail/chart/Para.tsx:59]: Each tab page you open holds its own data

File: ts/webui/src/components/experiment/trialdetail/ChangeColumnComponent.tsx [hunk 2]
```diff
@@ -67,9 +67,9 @@ class ChangeColumnComponent extends React.Component<ChangeColumnProps, ChangeCol
         const selectedColumns = allColumns.map(column => column.key).filter(key => currentSelected.includes(key));
         onSelectedChange(selectedColumns);
         if (whichComponent === 'table') {
-            localStorage.setItem('columns', JSON.stringify(selectedColumns));
+            localStorage.setItem(`${EXPERIMENT.profile.id}_columns`, JSON.stringify(selectedColumns));
```
Comment (ultmaster) [inline ts/webui/src/components/experiment/trialdetail/ChangeColumnComponent.tsx:70]: Can we set expire time to (for example) 1 month?

Comment (Lijiaoa) [reply ts/webui/src/components/experiment/trialdetail/ChangeColumnComponent.tsx:70]: 这个需要二次封装哦，localstorage本身没有过期时间。用户在console平台localstorage.removeItem也可以删

Comment (Lijiaoa) [reply ts/webui/src/components/experiment/trialdetail/ChangeColumnComponent.tsx]: just add it!

File: ts/webui/src/static/model/localStorage.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    value: string;
+}
+
+export const getValue = (key): null | string => {
+    const val = localStorage.getItem(key);
+    if (!val) {
+        return null;
+    }
+    const data = JSON.parse(val) as StorageFormat;
+    if (Date.now() - data.time > data.expire) {
```
Comment (ultmaster) [inline ts/webui/src/static/model/localStorage.ts:13]: This is not a correct expire. This is "expire on access". It means that it won't expire if nobody tries to access it.

Comment (Lijiaoa) [reply ts/webui/src/static/model/localStorage.ts:13]: @liuzhe-lz we plan to delete expire time in v2.9.1. Do you agree with us?

**Review by ultmaster — APPROVED** [review]

**Pull request merged**